core/worker.cc

```diff
@@ -403,20 +406,59 @@ bool detach_tc(bess::TrafficClass *c) {
   return remove_tc_from_orphan(c);
 }
 
+void run_global_resume_hooks() {
```

Comment:
`resume_hook.cc` might be a better place for this function (not sure)

Reply:
Agreed. I'll need to rework this slightly. Moving as is introduces an awkard circular dependency between `resume_hook.h` and `resume_hooks/setup_metadata.h`